src/api-explorer/v4-0/SpendUser.swagger2.json

```diff
@@ -858,30 +879,35 @@
               "$ref": "#/components/schemas/CustomData"
             }
           }
-        }
+        },
+        "description": "A user's base spend information"
       },
       "DelegateExtension": {
         "type": "object",
         "properties": {
           "expense": {
             "type": "array",
+            "description": "A list of expense delegates",
```

Comment:
Does the delegate extension only show 3 of 7 type?

Reply:
There are only 3 valid prod codes for delegation: `Expense`, `Payment` and `Purchase Request` at least given the code around the validation of the passed prod code.